docs/Update.md

```diff
@@ -1,27 +1,37 @@
 # How to update AzerothCore to the latest stable version
 
-## 1) Update your local sources
+## Verify that you're on the latest release
+
+1. Type `.server debug` in your console.
+  
+    1. If it outputs `Using World DB: ACDB 335.4-dev` then you can proceed to [Update your local source](#update-your-local-source).
```

Comment:
Maybe this PR isn't the right place for this, but I've run all the upgrade scripts and my version still says 335.3, The only spot I see a SQL statement to set the version is the initial insert statement in data/sql/base/db_world/version.sql, which I don't think is run during upgrades

Reply:
I usually drop my entire db when I update so I can't say :shrug:

Reply:
How do you keep your player's data intact?

Reply:
I only use it for Dev, but worldserver is disposable so I only drop that as it's rhe only one I usually edit to fix stuff. And then let the assambler update the char/auth dbs the few times they are updated.

Reply:
Does `world` just not store any state? Like I could drop that and recreate it whenever I want, even as say... An owner of a 100+ player server?

Reply:
World is designed to be disposable, only auth and char dbs hold player data